src/helpers/color-helpers.ts

```diff
@@ -0,0 +1,19 @@
+type ColorMethod = 'hex' | 'rgb' | 'rgba' | 'hsl' | 'hsla' | 'named';
+
+export const isValidColor = (color: string, method?: ColorMethod): boolean => {
+  const patterns: Record<ColorMethod, RegExp> = {
+    hex: /^#(?:[\dA-Fa-f]{3}){1,2}$/,
```

Comment:
Do we want to accept hex with transparency? In which case, the expression should likely be:
```suggestion
    hex: /^#(?:[\dA-Fa-f]{3,4}){1,2}$/,
```